shared-data/contrib/hints/hints/dragging-tags.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layouts/" + render_mode + "-tall.html" %}
+{% block title %}{{_("Dragging and dropping tags")}}{% endblock %}
+{% block content %}
+<div class="content-normal" style="max-width: 46em;">
+
+  <h1><span class="icon-tag"></span>
+    {{_("Dragging and dropping tags")}}
+  </h1>
+  <p>
+  	{{_("You can drag and drop tags or messages to add or change tags.")}}<br>
+  	{{_("The rules are following:")}}
+  </p>
+  <ul class="list" style="margin-left: 2em; list-style: disc;">
+  	<li>{{_("If you drag a message from a search result view onto a tag, all tags from the message are removed and the tag you drop the message on is added.")}}</li>
+  	<li>{{_("if you drag a tag from the sidebar onto a message, the tag is added to the message and other existing tags are not removed.")}}</li>
```

Comment:
I think it might be better to omit the word "existing" here - it's not the right word to describe the relationship between a tag and the messages it has been applied to. In general, the vocabulary here is tricky. Are tags added and removed? Are they applied? Are messages tagged and untagged? Sorry if this is nit-picky, but since tags are confusing to many people I think it's important that our descriptions are as clear as possible.